modules/nw-multus-host-device-object.adoc

```diff
@@ -43,6 +43,7 @@ following parameters: `device`, `hwaddr`, `kernelpath`, or `pciBusID`.
 The following object describes the configuration parameters for the host-device CNI
 plug-in:
 
+// containernetworking/plugins/.../host-device.go#L50
```

Comment:
It was intentional so that in the future, I'd know where to look for an API change without fumbling for too long.  If it's forbidden to leave clues, let me know.  Thanks again.

Reply:
Nope it's not, just wanted to check!